### PR TITLE
Use static methods maybe better

### PR DESCRIPTION
**Pull request by zhaoyuguang**
Avoid accessing static variables or static methods of this class through object references of a class. It increases the cost of compiler parsing and can be accessed directly by class name.

Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: config/src/main/java/com/alibaba/nacos/config/server/service/DynamicDataSource.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public ApplicationContext getApplicationContext() {
     public DataSourceService getDataSource() {
         DataSourceService dataSourceService = null;
 
-        if (STANDALONE_MODE && !propertyUtil.isStandaloneUseMysql()) {
+        if (STANDALONE_MODE && !PropertyUtil.isStandaloneUseMysql()) {
```
Comment (kezhenxu94) [inline config/src/main/java/com/alibaba/nacos/config/server/service/DynamicDataSource.java]: Then the `propertyUtil` member variable (line 35) could be removed?

File: config/src/main/java/com/alibaba/nacos/config/server/service/LocalDataSourceServiceImpl.java [hunk 2]
```diff
@@ -90,7 +90,7 @@ public void init() {
         tm.setDataSource(ds);
         tjt.setTimeout(5000);
 
-        if (STANDALONE_MODE && !propertyUtil.isStandaloneUseMysql()) {
```
Comment (kezhenxu94) [inline config/src/main/java/com/alibaba/nacos/config/server/service/LocalDataSourceServiceImpl.java]: Same as above

Comment (zhaoyuguang) [reply config/src/main/java/com/alibaba/nacos/config/server/service/LocalDataSourceServiceImpl.java]: ok

File: config/src/main/java/com/alibaba/nacos/config/server/service/DynamicDataSource.java (hunk 1 above)
Comment (zhaoyuguang) [reply config/src/main/java/com/alibaba/nacos/config/server/service/DynamicDataSource.java]: ok

**Comment by nkorange** [comment]
@kezhenxu94 Thanks for your advice.

**Comment by nkorange** [comment]
@zhaoyuguang Next time you should create an issue before you create a PR, and the target branch should be develop.

**Comment by zhaoyuguang** [comment]
> @zhaoyuguang Next time you should create an issue before you create a PR, and the target branch should be develop.

OK

**Pull request merged**